docker-compose.yml

```
@@ -51,6 +51,15 @@ services:
       options:
         max-size: "512m"
         max-file: "4"
+  solr_updater:
```

Most named resources in this file are named with hyphens; for consistency:

```suggestion
  solr-updater:
```